src/util.ts

```diff
@@ -9,6 +9,52 @@ export interface BabelProperty extends ESTree.MethodDefinition {
   decorators?: BabelDecorator[];
 }
 
+/**
+ * Returns if given node has a lit identifier
+ * @param {ESTree.Node} node
+ * @return {boolean}
+ */
+function hasLitIdentifier(
+  node:
+    | ESTree.Node
+    | ESTree.Expression
+    | ESTree.SpreadElement
+    | null
+    | undefined
+): boolean {
+  return node?.type === 'Identifier' && node?.name === 'LitElement';
+}
+
+/**
+ * Returns if the given node is a lit element by expression
+ * @param {ESTree.Node} node
+ * @return {boolean}
+ */
+function isLitByExpression(
+  node: ESTree.Node | ESTree.Expression | undefined | null
+): boolean {
+  if (hasLitIdentifier(node)) {
+    return true;
+  }
+  if (node?.type === 'CallExpression') {
+    return node.arguments?.some(isLitByExpression);
```

Comment:
im not sure how this built since `arguments?` implies the return type is `boolean | undefined` rather than `boolean`

maybe its because `node.arguments` isn't actually optional, so we can drop the `?`